src/libraries/System.Private.CoreLib/src/System/Reflection/SignatureConstructedGenericType.cs

```diff
@@ -50,6 +50,7 @@ public sealed override bool ContainsGenericParameters
             }
         }
 
+        protected sealed override bool IsValueTypeImpl() => _genericTypeDefinition.IsValueType;
```

Comment:
We should add tests to https://github.com/dotnet/runtime/blob/main/src/libraries/System.Runtime/tests/System.Runtime.Tests/System/Reflection/SignatureTypes.cs

Reply:
Sure - done